src/preset-themes/default.json

```diff
@@ -32,6 +32,11 @@
       "h": 0,
       "s": 0,
       "l": 100
+    },
+    "tab_line": {
+      "h": 360,
+      "s": 100,
+      "l": 60
```

Comment:
You'll need to tweak the /docs/theme-schema.json file as well. Along with all the other hsl[a] definitions, add the following new property:

```js
                "tab_line": {
                    "type": "object",
                    "properties": {
                        "h": {
                            "type": "integer",
                            "maximum": 360,
                            "minimum": 0
                        },
                        "s": {
                            "type": "integer",
                            "maximum": 100,
                            "minimum": 0
                        },
                        "l": {
                            "type": "integer",
                            "maximum": 100,
                            "minimum": 0
                        }
                    },
                    "required": ["h", "s", "l"],
                    "additionalProperties": false
                }
```

It's a bit of a funny one, since we've only added a `tab_line` definition to 2 of the 8 themes, so it isn't a required property unless we backfill it in to the other themes. But this should be enough to pass the super awesome theme linter.